data/campaigns/The_Rise_Of_Wesnoth/scenarios/12_A_Final_Spring.cfg

```diff
@@ -386,6 +386,31 @@
         [/move_unit_fake]
 
         {HERO (Merman Warrior) (Lord Typhon) ( _ "Lord Typhon") 1 3 20 ("portraits/typhon.png")}
+        # Always give Lord Typhon a storm trident. Object effects copied from data/core/macros/items.cfg
+        [+unit]
+            [+modifications]
+                [object]
+                    duration=forever
+                    [effect]
+                        apply_to=new_attack
+                        name="storm trident"
+                        description= _ "storm trident"
```

Comment:
If you want to backport in to 1.16 as well you need to add the textdomain used in items.cfg, else it will add a new translatable text whereas we have a string freeze at the moment.

Reply:
like
```
#textdomain wesnoth
                                                description= _ "storm trident"
#textdomain wesnoth-trow
```

Reply:
Totally forgot about that! good catch, thanks!